skillNer/general_params.py

```diff
@@ -1,19 +1,20 @@
 # native packs
-import os
 import json
 from posixpath import dirname
 # installed packs
 #
 # my packs
 from skillNer.network.remote_db import RemoteBucket
 
+# TODO needs to be updated to accommodate custom skill lists with different skill types
```

Comment:
I agree with you!

We will change that in this PR to enable custom support of skill databases